commands/init_templating.go

```diff
@@ -298,7 +298,7 @@ func getTemplates(brand platformsh.CloudBrand, rootDirectory, chosenTemplateName
 
 	templateFuncs := getTemplateFuncs(rootDirectory, minorPHPVersion)
 	var templates map[string]*template.Template
-	if brand == platformsh.UpsunBrand {
+	if brand == upsun.UpsunBrand {
```

Comment:
this reads weird 😄

Reply:
See https://github.com/symfony-cli/symfony-cli/pull/661